src/components/Tracker.js

```diff
@@ -1,18 +1,64 @@
 import React, {useEffect, useState} from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import {Link} from 'react-router-dom';
+import Button from 'react-bootstrap/Button';
 import Dropdown from 'react-bootstrap/Dropdown';
 import Container from "react-bootstrap/Container";
+import Form from 'react-bootstrap/Form';
 import Row from "react-bootstrap/Row";
 import Col from "react-bootstrap/Col";
 import Table from 'react-bootstrap/Table';
 import Pagination from 'react-bootstrap/Pagination';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faCheck, faTimes, faSortNumericDown, faSortNumericUp } from '@fortawesome/free-solid-svg-icons'
 import {searchXref} from '../actions/searchActions';
-import {searchMissingFiles, addWorkflowTag, setOrder, setTrackerPage} from '../actions/trackerActions';
+import {searchMissingFiles, addWorkflowTag, setOrder, setTrackerPage, setTrackerFilter} from '../actions/trackerActions';
 import LoadingOverlay from "./LoadingOverlay";
 
+const DownloadButton = (mod) => {
+  const [downloading, setDownloading] = useState(false);
+  const orderBy = useSelector(state => state.tracker.orderBy);
+  const trackerFilter = useSelector(state => state.tracker.trackerFilter);
+
+  const handleDownload = () => {
+    setDownloading(true);
+    let apiUrl = `${process.env.REACT_APP_RESTAPI}/reference/download_tracker_table/${mod.mod}?order_by=${orderBy}&filter=${trackerFilter}`;
```

Comment:
I thought mod was accessLevel which is a string, but if mod.mod works, then cool.

Reply:
Cool, thanks @pjhale